codecparsers/h265Parser.cpp

```diff
@@ -262,29 +291,29 @@ bool Parser::profileTierLevel(ProfileTierLevel* ptl, NalReader& nr,
         || ptl->general_profile_compatibility_flag[4]
         || ptl->general_profile_compatibility_flag[5])
         //The number of bits in this syntax structure is not affected by this condition
-        ptl->general_inbld_flag = nr.read(1);
+        READ(nr, ptl->general_inbld_flag);
     else
         nr.skip(1); // general_reserved_zero_bit
-    ptl->general_level_idc = nr.read(8);
+    READ_BITS(nr, ptl->general_level_idc, 8);
     for (uint32_t i = 0; i < maxNumSubLayersMinus1; i++) {
-        ptl->sub_layer_profile_present_flag[i] = nr.read(1);
-        ptl->sub_layer_level_present_flag[i] = nr.read(1);
+        READ(nr, ptl->sub_layer_profile_present_flag[i]);
+        READ(nr, ptl->sub_layer_level_present_flag[i]);
     }
     if (maxNumSubLayersMinus1 > 0) {
         for (uint32_t i = maxNumSubLayersMinus1; i < 8; i++)
             nr.skip(2); // reserved_zero_2bits
     }
     for (uint32_t i = 0; i < maxNumSubLayersMinus1; i++) {
         if (ptl->sub_layer_profile_present_flag[i]) {
-            ptl->sub_layer_profile_space[i] = nr.read(2);
-            ptl->sub_layer_tier_flag[i] = nr.read(1);
-            ptl->sub_layer_profile_idc[i] = nr.read(5);
+            READ_BITS(nr, ptl->sub_layer_profile_space[i], 2);
+            READ(nr, ptl->sub_layer_tier_flag[i]);
+            READ_BITS(nr, ptl->sub_layer_profile_idc[i], 5);
             for (uint32_t j = 0; j < 32; j++)
-                ptl->sub_layer_profile_compatibility_flag[i][j] = nr.read(1);
-            ptl->sub_layer_progressive_source_flag[i] = nr.read(1);
-            ptl->sub_layer_interlaced_source_flag[i] = nr.read(1);
-            ptl->sub_layer_non_packed_constraint_flag[i] = nr.read(1);
-            ptl->sub_layer_frame_only_constraint_flag[i] = nr.read(1);
+                READ(nr, ptl->sub_layer_profile_compatibility_flag[i][j]);
+            READ(nr, ptl->sub_layer_progressive_source_flag[i]);
+            READ(nr, ptl->sub_layer_interlaced_source_flag[i]);
+            READ(nr, ptl->sub_layer_non_packed_constraint_flag[i];
+			READ(nr, ptl->sub_layer_frame_only_constraint_flag[i]);
```

Comment:
@xuguangxin yes, i used git-clang-format. Does it support tab? Because in my editor, these lines are clean and tidy.